avocado.spec

```diff
@@ -91,6 +120,46 @@ arbitrary filesystem location.
 %files plugins-output-html
 %{python_sitelib}/avocado_result_html*
 
+%package plugins-runner-remote
+Summary: Avocado Runner for Remote Execution
+Requires: avocado == %{version}, fabric
+
+%description plugins-runner-remote
+Allows Avocado to run jobs on a remote machine, by means of an SSH
+connection.  Avocado should be previously installed on the remote machine.
+
+%files plugins-runner-remote
+%{python_sitelib}/avocado_runner_remote*
+
+%package plugins-runner-vm
+Summary: Avocado Runner for libvirt VM Execution
+Requires: avocado == %{version}, avocado-plugins-runner-remote == %{version}
+Requires: libvirt-python
+
+%description plugins-runner-vm
+Allows Avocado to run jobs on a libvirt based VM, by means of
+interaction with a libvirt daemon and an SSH connection to the VM
+itself.  Avocado should be previously installed on the VM.
+
+%files plugins-runner-vm
+%{python_sitelib}/avocado_runner_vm*
+
+%package plugins-runner-docker
+Summary: Avocado Runner for Execution on Docker Containers
+Requires: avocado == %{version}, avocado-plugins-runner-remote == %{version}
+Requires: aexpect
+# FIXME: add conditional dependency, since python-aexpect is still AFAIK
+# not available on EPEL.
```

Comment:
Basically this change (from this version):

```diff
diff --git a/avocado.spec b/avocado.spec
index 3deb5c6..2a70e0e 100644
--- a/avocado.spec
+++ b/avocado.spec
@@ -14,7 +14,12 @@ URL: http://avocado-framework.github.io/
 Source0: https://github.com/avocado-framework/%{name}/archive/%{commit}/%{name}-%{version}-%{shortcommit}.tar.gz
 BuildArch: noarch
 Requires: python, python-requests, pyliblzma, gdb, gdb-gdbserver, python-stevedore, python-yaml, procps-ng
-BuildRequires: python2-devel, python-setuptools, python-docutils, python-mock, python-psutil, python-sphinx, python-requests, aexpect, pystache, yum, python-stevedore, python-lxml, perl-Test-Harness, fabric, python-flexmock, libvirt-python, python-yaml, procps-ng
+BuildRequires: python2-devel, python-setuptools, python-docutils, python-mock, python-psutil, python-sphinx, python-requests, pystache, yum, python-stevedore, python-lxml, perl-Test-Harness, fabric, python-flexmock, libvirt-python, python-yaml, procps-ng
+%if 0%{?fedora}
+BuildRequires: python-aexpect
+%else
+BuildRequires: aexpect
+%endif
 
 %if 0%{?fedora} >= 25
 BuildRequires: kmod
@@ -147,9 +152,11 @@ itself.  Avocado should be previously installed on the VM.
 %package plugins-runner-docker
 Summary: Avocado Runner for Execution on Docker Containers
 Requires: avocado == %{version}, avocado-plugins-runner-remote == %{version}
+%if 0%{?fedora}
+Requires: python-aexpect
+%else
 Requires: aexpect
-# FIXME: add conditional dependency, since python-aexpect is still AFAIK
-# not available on EPEL.
+%endif
 
 %description plugins-runner-docker
 Allows Avocado to run jobs on a Docker container by interacting with a
```